subprojects/docs/src/snippets/tasks/incrementalBuild-customTaskClass/tests/customTaskClassWithInputOutputAnnotationsUpToDate.out

```diff
@@ -1,4 +1,4 @@
 > Task :processTemplates UP-TO-DATE
 
 BUILD SUCCESSFUL in 0s
-3 actionable tasks: 3 up-to-date
+1 actionable task: 1 up-to-date
```

Comment:
Here, I updated the expected outputs for the test [customTaskClassWithInputOutputAnnotations.sample.conf](https://github.com/gradle/gradle/blob/master/subprojects/docs/src/snippets/tasks/incrementalBuild-customTaskClass/tests/customTaskClassWithInputOutputAnnotations.sample.conf).

This change was required because in this test the task `processTemplates` has to be executed twice. 
Without configuration cache feature, two additional tasks coming from [buildSrc](https://scans.gradle.com/s/foqi3mkyjwzty/timeline?hide-timeline&outcome=from_cache,up_to_date&sort=longest) were executed along with `processTemplates`. 
However, with configuration cache, in the first execution of `processTemplates` the outputs from the `buildSrc`'s tasks are cached too and don't appear as executed (or up-to-date) in the second run.

Reply:
Sorry for the long delay on responding to this, @vmmaldonadoz. This poses a challenge to our testing framework, as there is no single expected output that will work with and without the configuration cache. We will are discussing what we could do here - disabling CC for the `processTemplates` task would be a possible workaround.